.github/workflows/build.yml

```diff
@@ -0,0 +1,18 @@
+name: CI
+on: [push]
+env:
+  NODE_OPTIONS: "--max_old_space_size=4096"
```

Comment:
I also wonder if we could get away without this, since here we only test the SDK.